### PR TITLE
HACKING.md: document SNAP_CLIENT_DEBUG_HTTP usage

### PR DESCRIPTION
**Pull request by miguelpires**
Document SNAP_CLIENT_DEBUG_HTTP in the HACKING.md document

### CODE REVIEW

**Review by mvo5 — APPROVED** [review]
Thank you!

**Comment by MiguelPires** [comment]
@degville hey, just pinging you in case you haven't seen this :slightly_smiling_face:

File: HACKING.md [hunk 1]
```diff
@@ -467,6 +467,10 @@ transfer it to the snappy system and then run:
 To debug interaction with the snap store, you can set `SNAPD_DEBUG_HTTP`.
 It is a bitfield: dump requests: 1, dump responses: 2, dump bodies: 4.
 
+Similarly, to debug the interaction between the `snap` command-line tool and the
+snapd's REST API, you can set `SNAP_CLIENT_DEBUG_HTTP`. It is also a bitfield,
+using the same values and behaviour as `SNAPD_DEBUG_HTTP`.
+
```
Comment (degville) [inline HACKING.md:473]:
```suggestion
Similarly, to debug the interaction between the `snap` command-line tool and the
snapd REST API, you can set `SNAP_CLIENT_DEBUG_HTTP`. It is also a bitfield,
with the same values and behaviour as `SNAPD_DEBUG_HTTP`.
```

**Review by degville — APPROVED** [review]
Looks good to me with tiny suggestions - thanks for pinging me @MiguelPires. I need to make checking our new labels part of my routine.

**Pull request merged**